rest/serializers.py

```diff
@@ -97,3 +103,49 @@ class Meta:
             'id',
             'slug',
         )
+
+
+class GroupSerializer(Serializer):
+    """
+    Serializer for base_group_type
+    """
+    # pylint: disable=abstract-method
+    group_type = ChoiceField(
+        choices=BaseGroupTypes.all_base_groups()
+    )
+
+    def validate_group_type(self, value):
+        """Validate group_type"""
+        # pylint: disable=no-self-use
+        if value not in BaseGroupTypes.all_base_groups():  # pragma: no cover
+            raise ValidationError(
+                'group "{group}" is not valid'.format(group=value)
+            )
+        return value
+
+
+class UserSerializer(Serializer):
+    """
+    Serializer for user
+    """
+    # pylint: disable=abstract-method
+    username = CharField()
+
+    def validate_username(self, value):
+        """Validate username"""
+        # pylint: disable=no-self-use
+        try:
+            User.objects.get(username=value)
+        except User.DoesNotExist:  # pragma: no cover
+            raise ValidationError(
+                'user "{user}" is not available'.format(user=value)
+            )
+        return value
+
+
+class UserGroupSerializer(UserSerializer, GroupSerializer):
+    """
+    Serializer for username base_group_type association
+    """
+    # pylint: disable=abstract-method
+    pass
```

Comment:
Actually, I can only if I remove the pylint disable, otherwise it will not be applied